CMakeLists.txt

```
@@ -167,3 +160,24 @@ function(mbed_generate_bin_hex target)
         VERBATIM
     )
 endfunction()
+
+#
+# Configure Mbed OS CMake application (default).
+#
+function(mbed_configure_application app_target)
```

```suggestion
function(mbed_configure_application target)
```
The prefix is not necessary as the function already says it is to configure the application.
Similar comment regarding the argument name for `mbed_configure_component()` below.

---

both suggestions applied